src/data/faq_de.json

```diff
@@ -435,6 +435,17 @@
                             "Nein, diese sind nicht Bestandteil der Datensicherung. Das heißt, die gesammelten Zufalls-IDs können auch nicht auf dem selben oder einem anderen Gerät wiederhergestellt werden."
                         ]
                     },
+                    {
+                        "title": "Was muss ich beim Wechsel auf ein neues Gerät beachten?",
+                        "anchor": "new_device",
+                        "active": true,
+                        "textblock": [
+                            "Ein Backup der alten Zufalls-IDs ist leider nicht möglich, da die Expusure Notification API von Apple/Google keinen Export dieser Daten erlaubt. Damit Sie trotzdem über mögliche Risikobegegnungen informiert werden, empfehlen wir Ihnen, den Risikostatus auf dem alten Gerät noch mindestens 14 Tage zu beobachten, nachdem Sie die Corona-Warn-App auf dem neuen Gerät installiert haben. Beachten Sie dazu folgende Hinweise:",
+                            "<ul><li>Ihr altes Gerät muss in der Lage sein, regelmäßige Überprüfungen auf mögliche Begegnungen durchzuführen. Lassen Sie das Gerät daher permanent eingeschaltet und dauerhaft mit dem Internet verbunden.</li><li>Bewahren Sie das alte Gerät an einem stationären Ort auf, z.B. an einem bestimmten Platz in Ihrer Wohnung. Falls möglich, platzieren Sie das Gerät in Hör- oder Sichtweite, sodass Sie bei einer Benachrichtigung der App frühzeitig gewarnt werden.</li><li>Rufen Sie die Corona-Warn-App auf dem alten Gerät mindestens einmal täglich aktiv auf und überprüfen Sie dort Ihren Risikostatus.</li></ul>",
+                            "Nach 14 Tagen enthält Ihr altes Gerät keine relevanten Einträge für die Risiko-Ermittlung mehr. Sie brauchen den Risikostatus auf diesem Gerät dann nicht weiter zu beachten und können die Corona-Warn-App dort entfernen.",
+                            "Falls Sie innerhalb der 14 Tage auf COVID-19 gestestet wurden, tragen Sie den QR-Code auf Ihrem neuen Gerät ein. Ist das Testergebnis positiv, sollten Sie das Testergebnis auf beiden Geräten teilen. Für das Teilen eines Testergebnisses auf mehreren Geräten beachten Sie bitte <a href='https://www.coronawarn.app/de/faq/#test_multiple_devices' target='_blank' rel='noopener noreferrer'>diesen</a> FAQ Eintrag."
```

Comment:
<a href='https://www.coronawarn.app/de/faq/#test_multiple_devices' target='_blank' rel='noopener noreferrer'>diesen</a>

Preferable to use relative link #test_multiple_devices instead of absolute link https://www.coronawarn.app/de/faq/#test_multiple_devices